src/components/Dashboard/TodoList.vue

```diff
@@ -877,7 +878,7 @@ export default class TodoList extends Mixins(
         name: FilingTypes.DISSOLUTION,
         filingId: header.filingId,
         legalType: corpTypeDescription,
-        title: 'Voluntary Dissolution',
+        title: this.isFirm ? 'Dissolution' : 'Voluntary Dissolution',
```

Comment:
1. I'd really prefer to see the resources used here instead of making this component specific for firms.

2. It's odd that this displays "Dissolution" for a firm and "VD" for everything else, but the resources specify the exact opposite. Typo?

Reply:
Awesome! Thanks.